README.md

```diff
@@ -154,8 +158,7 @@ To execute a TTG we must make it executable (this will declare the TTG  program
       tt->invoke();
 ```
 
-`ttg::execute()` must occur before, not after, sending any messages. Note also that we must ensure that only one such message must be generated. Since TTG execution uses the Single Program Multiple Data (SPMD) model,
-when launching the TTG program as multiple processes only the first process (rank) gets to send the message.
+`ttg::execute()` must occur before, not after, sending any messages. Note also that we must ensure that only one such message is generated. Since TTG execution uses the Single Program Multiple Data (SPMD) model, when launching the TTG program as multiple processes only the first process (rank) gets to send the message.
```

Comment:
```suggestion
`ttg::execute()` starts the TTG work engines and must occur before, not after, kickstarting off any computation by sending the data into the free in-terminals. The kickstart messages are generated by invoking `TT::invoke(taskID, values)`; in this case the task ID is void, and the task does not take any input data, so `TT::invoke()` receivesno parameters. Since TTG uses the Single Program Multiple Data (SPMD) execution model, every process invokes `ttg::execute()` when this TTG program is invoked as multiple processes, but only the first process (rank) gets to send the kickstart message since all messages in a TTG program, including the kickstart messages, must be unique.
```